card/mod/machines/set/type/css.rb

```diff
@@ -55,6 +55,12 @@ def default_nest_view
   view :content_changes, mod: Abstract::Script::HtmlFormat
 end
 
+format :css do
+  view :import do
+    %{\n@import url("#{_render_url}");\n}
+  end
+end
```

Comment:
This is definitely needed and correct. That's were the nesting ends. But the other import views do unnecessary stuff.